pkg/cmd/roachtest/roachtestutil/clusterupgrade/clusterupgrade.go

```diff
@@ -87,6 +87,11 @@ func (v *Version) AtLeast(other *Version) bool {
 	return v.Version.AtLeast(&other.Version)
 }
 
+// Series returns the release series this version is a part of.
+func (v *Version) Series() string {
```

Comment:
Duplicates `versionSeries` in `release`. Maybe we should just make the latter public and reuse?

Reply:
Done.